src/app/styles/bootstrap/components/_buttons.scss

```diff
@@ -0,0 +1,34 @@
+@mixin button($property) {
```

Comment:
ok, it's unclear to me on a casual read how much is being over-ridden here.

Are we totally replacing the bootstrap `button` mixin?

Reply:
We are not modifying the bootstrap mixin at all, this has no reference to the way bootstrap sets the button properties, and instead just overrides the end styling. Its effectively just adding styling on top of the bootstrap styling similar to the following:

```css
.btn-primary {
  // bootstrap styling
}

.btn-primary {
  // custom styling
}